### DISCUSSION TITLE
Migrating an existing instance to an object store

### DISCUSSION THREAD

**Discussion by AmaranthineCodices** [TandoorRecipes/recipes#2009] [Q&A]
Hi,

When I set up my Tandoor instance I didn't configure an object store like S3 for media files. My host machine is somewhat limited in disk space, so in the future I expect to need to switch over to an object store as my user base uploads files. How can I migrate an existing instance, including all the media files uploaded up to that point, to an object store? Is there a migration path?

Thanks!

**Comment by vabene1111** [comment]
I honestly have no idea 🤣

But I am pretty sure that you could just change the config to s3, add one image so that the paths are created and then just copy all of the data from the old folder to the new object storage keeping the paths the same.

**Comment by vabene1111** [comment]
Make sure to let me know how the migration went.
